docs/core/whats-new/dotnet-core-2-1.md

```diff
@@ -90,24 +90,31 @@ All .NET Core applications starting with the .NET Core 2.0 automatically roll fo
 Starting with .NET Core 2.0, if the version of .NET Core that an application was built with is not present at runtime, the application automatically runs against the latest installed *minor version* of .NET Core. In other words, if an application is built with .NET Core 2.0, and .NET Core 2.0 is not present on the host system but .NET Core 2.1 is, the application runs with .NET Core 2.1.
 
 > [!IMPORTANT]
-> This roll-forward behavior doesn't apply to preview releases. Nor does it apply to major releases. For example, a .NET Core 1.0 application wouldn't roll forward to .NET Core 2.0 or .NET Core 2.1.
+> This roll-forward behavior doesn't apply to preview releases. By default it also doesn't apply to major releases, but this can be changed with the settings below.
 
-You can also disable minor version roll forward in any of three ways:
+You can modify this behavior by changing the setting for the roll-forward on no candidate shared framework. The available settings are:
```

Comment:
The meaning should be "general" since the setting can be applied in multiple ways, only one of which is tied to a specific shared framework. The env. variable and the command line switch are "global" and apply to all shared frameworks. That's why I tried to word it without mentioning any of the specific ways to modify the setting.